docker-compose.yml

```diff
@@ -45,3 +45,25 @@ services:
     depends_on:
       - api
     restart: "unless-stopped"
+
+  redis:
```

Comment:
Do we also want the app to halt if redis is down/broken for both prod and dev? I think we panic if the db isn't running when bootstrapping the backend, do we want the same behavior for redis? Just wondering.

Reply:
I'll address that in a follow-up PR.